components/data-display/album-preview-grid.tsx

```diff
@@ -12,6 +12,8 @@ export const AlbumPreviewGrid: FC<{
     gap="md"
     overflow="auto"
     pt="sm"
+    width={{base:"600px",md:"100%"}}
```

Comment:
```suggestion
    width={{base:"2xl",md:"full"}}
```

@nakata0047 

固定値ではなく近いトークンの値を使用してください。

https://yamada-ui.com/styled-system/theming/default-theme#sizes